mesonbuild/templates/valatemplates.py

```diff
@@ -84,7 +89,7 @@
 
 
 class ValaProject(SampleImpl):
-    def __init__(self, options):
+    def __init__(self, options: Arguments) -> None:
```

Comment:
Nitpicking: I think you don't need the explicit `-> None` and you didn't add it to the other `__init__()` methods.